api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java

```diff
@@ -216,12 +216,12 @@ public FeeDto toFeeDto(PaymentFee fee) {
     @SneakyThrows(NoSuchMethodException.class)
     private PaymentDto.LinkDto cancellationLink(String userId, Integer paymentId) {
         Method method = CreditAccountPaymentController.class.getMethod("cancel", String.class, Integer.class);
-        return new PaymentDto.LinkDto(ControllerLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
+        return new PaymentDto.LinkDto(WebMvcLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
```

Comment:
Codecov is complaining this line is not covered. Can you take a look?

Reply:
Hi @YasharRahvar - This is again a same issue that we faced during apportionment. We are good with the overall coverage and it is not going down.

Reply:
ok fine Aravin